org.BleuBleu.FamiStudio.yml

```diff
@@ -0,0 +1,65 @@
+app-id: org.BleuBleu.FamiStudio
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.mono6
+command: famistudio.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+# Read and write projects
+  - --filesystem=home
+# For the Mono runtime
+  - --env=MONO_GAC_PREFIX=${FLATPAK_DEST}
+# Midi keyboard input
+  - --device=all
+modules:
+  - shared-modules/gtk2/gtk2.json
+  - name: mono6
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/mono6/install.sh
+  - name: gtk-sharp2
+    buildsystem: simple
+    build-commands:
+      - export PATH=$PATH:/usr/lib/sdk/mono6/bin && export LD_LIBRARY_PATH=${LD_LIBRARY_PATH:+$LD_LIBRARY_PATH:}/usr/lib/sdk/mono6/lib && export PKG_CONFIG_PATH=${PKG_CONFIG_PATH:+$PKG_CONFIG_PATH:}/usr/lib/sdk/mono6/lib/pkgconfig && ./configure --prefix=/app && make && make install
+    sources:
+      - type: archive
+        url: https://download.mono-project.com/sources/gtk-sharp212/gtk-sharp-2.12.45.tar.gz
+        sha256: 02680578e4535441064aac21d33315daa009d742cab8098ac8b2749d86fffb6a
+      - type: patch
+        path: disambiguate-system.range-and-gtk.range.patch
+  - name: extralibs
+    buildsystem: simple
+    build-commands:
+      - cp -ar /usr/lib/sdk/mono6/lib/mono/gac/System.Drawing /app/lib/mono/gac/ && rm -f /app/lib/mono/gac/System.Drawing/*/*.pdb
+      - cp -ar /usr/lib/sdk/mono6/lib/mono/gac/System.Windows.Forms /app/lib/mono/gac/ && rm -f /app/lib/mono/gac/System.Windows.Forms/*/*.pdb
+  - name: app
+    buildsystem: simple
+    build-commands:
+      - install -d /app/bin/
+      - cp -r * /app/bin/
+      - ln -sf /usr/lib/x86_64-linux-gnu/libopenal.so.1 /app/bin/libopenal32.so
+    sources:
+      - type: archive
+        url: https://github.com/BleuBleu/FamiStudio/releases/download/2.2.0/FamiStudio220-LinuxAMD64.zip
+        sha256: 56e1e1c0a31c61a12b657320d976f5c6b85e6ebb3b0a5b38a36460d583e4bf9f
+        strip-components: 0
+  - name: extrafiles
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 famistudio.sh ${FLATPAK_DEST}/bin/famistudio.sh
+      - install -Dm644 org.BleuBleu.FamiStudio.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/org.BleuBleu.FamiStudio.svg
+      - install -Dm644 org.BleuBleu.FamiStudio.desktop ${FLATPAK_DEST}/share/applications/org.BleuBleu.FamiStudio.desktop
+      - install -Dm644 org.BleuBleu.FamiStudio.appdata.xml ${FLATPAK_DEST}/share/metainfo/org.BleuBleu.FamiStudio.appdata.xml
+    sources:
+      - type: file
+        path: famistudio.sh
+      - type: file
+        path: org.BleuBleu.FamiStudio.svg
+      - type: file
+        path: org.BleuBleu.FamiStudio.desktop
+      - type: file
+        path: org.BleuBleu.FamiStudio.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html